src/main/java/org/primefaces/util/ComponentUtils.java

```diff
@@ -171,10 +171,15 @@ public static Converter getConverter(FacesContext context, UIComponent component
         return context.getApplication().createConverter(converterType);
     }
 
-    public static Object getConvertedValue(FacesContext context, UIComponent component, String value) {
-        Converter converter = getConverter(context, component);
+    public static Object getConvertedValue(FacesContext context, UIComponent component, Object value) {
+        String submittedValue = null;
```

Comment:
You could use `String submittedValue = Objects.toString(value, null);` and also, make the change for every caller? (e.g EditorRenderer, SelectCheckboxMenu, etc.)